sdk/core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -228,6 +228,7 @@ public RuntimeException logExceptionAsWarning(RuntimeException runtimeException)
      * @return The passed {@link Throwable}.
      * @throws NullPointerException If {@code throwable} is {@code null}.
      */
+    @Deprecated
```

Comment:
You didn't add @deprecated to the javadoc

Reply:
Done